src/search/SearchMessagesScreen.js

```diff
@@ -49,6 +49,9 @@ class SearchMessagesScreen extends PureComponent<Props, State> {
       0,
       false,
     );
+
+    // TODO: on rebasing atop #3868, fetchMessagesComplete etc. will go here
```

Comment:
Hmm, I don't see anything called fetchMessagesComplete in the current PR or in #3868; where does this come from?

Reply:
Hmm, not sure. I may have just meant `messageFetchComplete`.

If not that, it would have been some kind of refactored version of that that I may have had in mind after our conversation about #3868. Fortunately this is a WIP note to myself (in this tip commit of the branch); I wouldn't want to merge something that elliptical :-)

Reply:
Haha, makes sense.

Reply:
Edited to `messageFetchComplete`. If I did have some refactor in mind, the "etc." can encompass that. :wink: